typeclasses/characters.py

```diff
@@ -9,10 +9,11 @@
 """
 # import importlib
 from evennia import DefaultCharacter
+from evennia.utils import lazy_property
 from utils.trait import Trait
```

Comment:
I know, I will update my repo and update my pull request
Related question: I put equip.py in world, should this be in utils? The handler is quite modular and not strictly related to the rpg system.